### PR TITLE
[RHOAIENG-11527] - [backend] Incorrect values for number of requests …

### PR DESCRIPTION
**Pull request by spolti**
…in Kserve Performance metrics
A summarized decision:
```
Update the chart title to match the approved by UXD. Use "Requests rate per 5 min" the same way as the model mesh to avoid confusion and maintain consistency as it denotes rate instead concrete values.

```

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: controllers/constants/runtime-metrics.go [hunk 1]
```diff
@@ -15,21 +15,23 @@ limitations under the License.
 
 package constants
 
+// ${DASHBOARD_RATE_INTERVAL} is a placeholder for the metrics that needs to receive values from the dashboard at the
```
Comment (terrytangyuan) [inline controllers/constants/runtime-metrics.go:18]: This should be `${REQUEST_RATE_INTERVAL}`?

Comment (spolti) [reply controllers/constants/runtime-metrics.go:18]: actually this comment needs to be removed, thanks for the heads up.

Comment (spolti) [reply controllers/constants/runtime-metrics.go]: updated, thanks

**Review by israel-hdez — APPROVED** [review]
/lgtm

We are moving forward this PR. This part is working fine, but there seems to be an issue in the dashboard leading to a bad chart.

**Pull request merged**